docs/release-notes/next-ceres/announcement.md

```diff
@@ -0,0 +1,75 @@
+Ceres is the 5th protocol upgrade for aeternity; following the initial protocol Roma, we have then seen Minerva, Fortuna, Lima, and Iris. Ceres is the first protocol upgrade not part of the initial update schedule and it contains mainly smaller improvements and enhancements.
```

Comment:
Possibly here at this one sentence to make sure people know where to look for details:
```suggestion
Ceres is the 5th protocol upgrade for aeternity; following the initial protocol Roma, we have then seen Minerva, Fortuna, Lima, and Iris. Ceres is the first protocol upgrade not part of the initial update schedule and it contains mainly smaller improvements and enhancements. The protocol documentation can be found at  https://docs.aeternity.com/protocol/
```

Reply:
Some of the formatting there is horrible but otherwise that is a good idea 👍